### PR TITLE
trip requests. account can accept or decline it 

### PR DESCRIPTION
**Pull request by taskiukas**
trip requests. account can accept or decline it

### CODE REVIEW

File: src/main/java/com/javainiaisuzspringom/tripperis/controllers/TripRequestController.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    @Autowired
+    TripRequestService tripRequestService;
+
+    @GetMapping("/api/myPendingRequests")
+    public List<TripRequest> currentUser(@AuthenticationPrincipal UserDetails userDetails) {
+        return tripRequestService.getMyPendingRequests(userDetails);
+    }
+
+    @GetMapping("/api/tripRequests/{id}")
+    public List<TripRequest> getTripRequestsByTripId(@AuthenticationPrincipal UserDetails userDetails, @PathVariable Integer id) {
```
Comment (pauliuskalv) [inline src/main/java/com/javainiaisuzspringom/tripperis/controllers/TripRequestController.java]: Gal perkeliam sita i TripController?

File: src/main/java/com/javainiaisuzspringom/tripperis/domain/TripRequestPatchDTO.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package com.javainiaisuzspringom.tripperis.domain;
+
+public class TripRequestPatchDTO {
```
Comment (pauliuskalv) [inline src/main/java/com/javainiaisuzspringom/tripperis/domain/TripRequestPatchDTO.java]: Gal tiesiog naudojam @Data Lombok anotacija? Sumazins kodo

Comment (taskiukas) [reply src/main/java/com/javainiaisuzspringom/tripperis/domain/TripRequestPatchDTO.java]: done

File: src/main/java/com/javainiaisuzspringom/tripperis/controllers/TripRequestController.java (hunk 1 above)
Comment (taskiukas) [reply src/main/java/com/javainiaisuzspringom/tripperis/controllers/TripRequestController.java]: done

**Pull request merged**